minimint-api/Cargo.toml

```diff
@@ -8,14 +8,13 @@ edition = "2018"
 [dependencies]
 anyhow = "1.0.58"
 async-trait = "0.1"
-bitcoin = { version = "0.28.1", features = [ "rand", "serde" ] }
+bitcoin = { version = "0.28.1", features = [ "rand", "serde", "use-serde"] }
```

Comment:
you can remove `"serde"` because use-serde enables that

Reply:
FYI `use-serde` is being renamed to `serde` in the next release https://github.com/rust-bitcoin/rust-bitcoin/pull/1006